components/experiment.tsx

```diff
@@ -1,69 +1,61 @@
-import styled from '@emotion/styled';
+'use client';
 import { Codepen, ExternalLink, GitHub } from 'react-feather';
-import { IExperiments } from '../content/experiments';
-import {
-  StyledExperimentItemContainer,
-  Tilter,
-} from './styles/experiments.styles';
+import Tilt from 'react-parallax-tilt';
 
-type Props = {
-  experiment: IExperiments;
+type Experiment = {
+  date: string;
+  description: string;
+  link: string;
+  location: string;
+  tags: string[];
+  title: string;
 };
 
-const GithubIcon = styled(GitHub)`
-  height: 0.8em;
-  width: 0.8em;
-  color: #777;
-`;
-
-const CodepenIcon = styled(Codepen)`
-  height: 0.8em;
-  width: 0.8em;
-  color: #777;
-`;
-const ExternalLinkIcon = styled(ExternalLink)`
-  height: 0.8em;
-  width: 0.8em;
-  color: #777;
-`;
-
-const ExperimentItem = ({ experiment }: Props) => {
+export function ExperimentItem({ experiment }: { experiment: Experiment }) {
```

Comment:
Added new created type from line 9 to avoid `any`

Reply:
created a PR in upstream
- https://github.com/vickonrails/next-starter-peacock/pull/25